.travis.yml

```diff
@@ -15,3 +15,4 @@ script:
   - ./vendor/bin/phpstan analyse --no-interaction --no-progress
   - ./vendor/bin/psalm
   - ./vendor/bin/infection
+  - ./vendor/bin/roave-backward-compatibility-check --from=1.0.1
```

Comment:
Note: this will make `master` red until we tag

Reply:
Ha, yes. Maybe we could add the bc check in a minor instead? Like, 2.1.0? I think that would make things cleaner. We'd have a green master and could go without `--from=1.0.1` from the beginning as well.

Reply:
We can also use ` || true` and use the output to generate release notes

Reply:
Does this feel clean to you? I'd rather generate them locally for now and then add the command to Travis as soon as we've tagged 2.0.0, but I'm not sure. Copying over the notes from Travis to the release or from the local command line to release doesn't really make a difference, or would you generate them completely automatically?

Reply:
Pushing that back to after 2.0.0 release as discussed.